mantidimaging/gui/windows/operations/presenter.py

```diff
@@ -120,19 +127,54 @@ def do_apply_filter_to_all(self):
         if not confirmed:
             return
         stacks = self.main_window.get_all_stack_visualisers()
+        if self.view.safeApply.isChecked():
+            self.original_images_stack = []
+            for stack in stacks:
+                self.original_images_stack.append((stack.presenter.images.copy(), stack.uuid))
```

Comment:
The with `operation_in_progress` should also be done here

Reply:
Done!